pkg/apis/kubermatic/v1/datacenter.go

```diff
@@ -693,6 +693,11 @@ type MeteringReportConfiguration struct {
 
 	// Interval defines the number of days consulted in the metering report.
 	Interval int `json:"interval,omitempty"`
+
+	// +optional
+
+	// Retention defines the number of days after which reports are queued for removal. The default value of "0" keeps reports forever.
+	Retention int `json:"retention,omitempty"`
```

Comment:
Can we add a validation rule to ensure this is never negative? Maybe there's some kubebuilder validation annotation thingy?